pkg/config/node.go

```diff
@@ -36,13 +36,13 @@ func (c *Config) CanonicalNodeName() string {
 }
 
 // Read or set the NodeName that will be used for this MicroShift instance
-func (c *Config) establishNodeName() (string, error) {
+func (c *Config) establishNodeName(dataDir string) (string, error) {
 	name := c.CanonicalNodeName()
-	filePath := filepath.Join(GetDataDir(), ".nodename")
+	filePath := filepath.Join(dataDir, ".nodename")
 	contents, err := os.ReadFile(filePath)
 	if os.IsNotExist(err) {
 		// ensure that dataDir exists
-		os.MkdirAll(GetDataDir(), 0700)
+		os.MkdirAll(DataDir, 0700)
```

Comment:
This is looking great, small nit, I think this needs to be the lowercase `dataDir`.

Reply:
good eye, yes, you're right